### PR TITLE
chore: upgrade to ng @ 9

### PR DESCRIPTION
**Pull request by vladjerca**
TODO:

- [x] investigate test failures

### CODE REVIEW

File: projects/angular/components/ui-grid/src/ui-grid.component.html [hunk 1]
```diff
@@ -240,7 +240,7 @@
         </div>
     </div>
     <div class="ui-grid-footer-container">
-        <mat-paginator *ngIf="!!footer"
+        <mat-paginator *ngIf="!!footer as footer"
```
Comment (sergiubologa) [inline projects/angular/components/ui-grid/src/ui-grid.component.html]: Why is `as` needed?

File: projects/angular/directives/ui-ng-let/src/ui-ng-let.directive.spec.ts [hunk 2]
```diff
@@ -80,9 +80,7 @@ describe('Directive: UiClipboard', () => {
         `,
     })
     class NgLetDynamicFixtureComponent {
-        @ViewChild('child', {
-            static: false,
```
Comment (sergiubologa) [inline projects/angular/directives/ui-ng-let/src/ui-ng-let.directive.spec.ts]: So `static` is not required anymore?

File: projects/angular/components/ui-grid/src/ui-grid.component.html (hunk 1 above)
Comment (vladjerca) [reply projects/angular/components/ui-grid/src/ui-grid.component.html]: It's nonsense, I've removed it. Ignore it I was really tired when porting 😬

File: projects/angular/directives/ui-ng-let/src/ui-ng-let.directive.spec.ts (hunk 2 above)
Comment (vladjerca) [reply projects/angular/directives/ui-ng-let/src/ui-ng-let.directive.spec.ts]: No, it now defaults to `false`.

**Comment by vladjerca** [comment]
@toxik @anbalase @sergiubologa Should be good to go.

File: projects/angular/directives/ui-virtual-scroll-range-loader/src/ui-virtual-scroll-range-loader.directive.spec.ts [hunk 3]
```diff
@@ -236,7 +236,7 @@ describe('Directive: UiVirtualScrollRangeLoaderDirective', () => {
             }),
         );
 
-        it('should emit  RangeLoad without the last item index if Pula that is loaded',
```
Comment (sergiubologa) [inline projects/angular/directives/ui-virtual-scroll-range-loader/src/ui-virtual-scroll-range-loader.directive.spec.ts]: Should we search for `Pula` in the entire lib? 🤣 🤣 🤣

**Pull request merged**